[docs] [PLAT-7426] added http/https proxy setup (#17600)
yugabyte/yugabyte-db: docs/content/v2.16/yugabyte-platform/install-yugabyte-platform/install-software/default.md

![Enable Proxy](/images/replicated/enable-proxy.png)

When completing the **Enable Proxy** settings, keep in mind the following:

- If your proxy is using the default ports for each protocol, then set the ports for the HTTP and HTTPS proxies to the default, 80 and 443 respectively, instead of 8080 and 8443 as shown in the preceding illustration.

- If you have only one proxy server set up (for both HTTP and/or HTTPS), then specify that proxy server for both the HTTP settings (the first three settings) and the HTTPS settings (settings four to six).

- The help text for each field shows which Java system property or environment variable gets set by the field. System properties have the "-D" prefix. For example "Specify -Dhttps.proxyPort".

- The **no proxy** fields (HTTP no proxy setting, HTTP no proxy setting for Java) are lists of exception hosts, provided as a comma-delimited list of addresses or hostnames. Include the following addresses:
  - The Docker gateway address (172.17.0.1 by default).
  - The address of any previously-specified web proxy.
  - Any other IP addresses that you deem safe to bypass the proxy.

> - These settings comprehensively govern all network connections that YBA initiates. For example, suppose that you specify a proxy server for HTTP, all unencrypted connections initiated by YBA will be affected. If you want YBA to connect to the database universe nodes bypassing the proxy server, then you must explicitly specify the database universe nodes' IP addresses as exception hosts (also known as "no proxy").

Reviewer: ```suggestion
- These settings comprehensively govern all network connections that YBA initiates. For example, if you specify a proxy server for HTTP, all unencrypted connections initiated by YBA will be affected. If you want YBA to bypass the proxy server when connecting to database universe nodes, then you must explicitly specify the database universe node IP addresses as exception hosts (also known as "no proxy").
```

> - Some of YBA's network connections are driven by YBA's Java process, while others are driven outside of Java (For example, via Python or a Linux shell execution). As such, there are separate parameters configurable described further.

Reviewer: ```suggestion
- Because some YBA network connections are driven by YBA's Java process, while others are driven outside of Java (for example, via Python or a Linux shell execution), each (Java and non-Java) has its own separate configurable parameters.
```
